framework/src/main/java/org/checkerframework/common/basetype/messages.properties

```diff
@@ -96,6 +97,11 @@ contracts.conditional.postcondition.false.methodref.invalid=Conditional postcond
 lambda.unimplemented=This version of the Checker Framework does not type-check lambda expressions.
 methodref.inference.unimplemented=This version of the Checker Framework does not type-check method references with implicit type arguments.
 
+invalid.polymorphic.qualifier.use=Cannot use polymorphic qualifier on a field unless the class is annotated with @HasQualifierParameter
```

Comment:
Wouldn't it be more useful to include the specific polymorphic qualifier? The field might be annotated with two different poly quals, and only one is invalid.